salt/master.py

```diff
@@ -1132,6 +1145,24 @@ def run(self):
         Start a Master Worker
         '''
         salt.utils.process.appendproctitle(self.name)
+
+        # if we inherit req_server level without our own, reset it
+        if not salt.utils.platform.is_windows():
+            enforce_mworker_niceness = True
+            if self.opts['req_server_niceness']:
+                if salt.utils.user.get_user() == 'root':
+                    log.info('%s decrementing inherited ReqServer niceness to 0', self.name)
+                    log.info(os.nice())
```

Comment:
A log call with just the `os.nice()` on the middle of a bunch of logs makes the logged information un-useful.
Either remove this call or join it with the debug call before:
`log.info('%s decrementing inherited ReqServer niceness to 0. Previous value: %s', self.name, os.nice())`

Reply:
this was an accidental include actually, there doesnt seem to be a way to get 'current' nicess. ive removed that log.info(os.nice()) call